scripts/ci

```diff
@@ -135,13 +135,13 @@ test_failed()
 # Background to start the Docker pulls
 ./test-warm >/dev/null &
 
-CI=true MAVEN_ARGS='-B -q' run ./build
-
 threads=$(($(nproc) + 1))
 if [ $threads -gt 8 ]; then
     threads=8
 fi
 
+CI=true MAVEN_ARGS="-B -q -T $threads" run ./build
```

Comment:
Ya, seems to be better now. The multi threading warnings are gone during the build. It hasn't tripped up yet. It does deadlock nicely with checkstyle though.